src/@orb.yml

```diff
@@ -4,19 +4,19 @@ description: Bundle of utils for GitHub pull requests
 examples:
   test-pull-request:
     description: |
-      Run pull request tests on a temporary branch resulting from merging the PR's head branch into the base branch, and send a comment on test failure.
+      Run pull request tests on a temporary branch resulting from merging the PR's head branch into the base branch. Post a PR comment on test failure, but Slack the author on success.
     usage:
       version: 2.1
       orbs:
-        jq: circleci/jq@1.9.0
-        gh-pr: narrativescience/gh-pr@0.0.3
+        ghpr: narrativescience/ghpr@x.y.z
       jobs:
         run-test:
-          docker:
-            - image: circleci/python:3.7.1
+          machine: true
```

Comment:
Can you explain why running on a machine is necessary now vs a container previously? Just curious and want to know more.

Reply:
`machine` is lighter weight that pulling down and building an image! `jq` is also pre-installed :)

Reply:
Oh nice, thanks!